Irina/src/main/java/oop/User.java

```diff
@@ -0,0 +1,14 @@
+package oop;
+
+public class User extends AbstractUser {
+    private final String username;
```

Comment:
You already have field name in parent class, why not to use it?

Reply:
Done